src/core/exception.d

```diff
@@ -855,8 +878,25 @@ extern (C)
     }
 }
 
+private size_t maxStoreSize()
+{
+    size_t size;
+    foreach (s; [
+        __traits(classInstanceSize, AssertError),
+        __traits(classInstanceSize, RangeError),
+        __traits(classInstanceSize, SwitchError),
+        __traits(classInstanceSize, ForkError),
+        __traits(classInstanceSize, InvalidMemoryOperationError),
+        __traits(classInstanceSize, OutOfMemoryError),
+        __traits(classInstanceSize, FinalizeError),
+        __traits(classInstanceSize, ArraySliceError),
```

Comment:
Shouldn't this list include `ArrayIndexError`?

Reply:
Done.